server/views/pages/partials/printNotification/sentenceTable.njk

```diff
@@ -0,0 +1,38 @@
+{% from "./sentenceLength.njk" import sentenceLength %}
+{% macro sentenceTable(model) %}
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-2">Sentences</h2>
+    <table class="govuk-table govuk-!-margin-bottom-4">
+      <caption class="govuk-table__caption govuk-table__caption--m govuk-visually-hidden" data-qa="sentence-title">Sentence details</caption>
+      <thead class="govuk-table__head ">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header" data-qa="sentence-col-title">Sentence</th>
+          <th scope="col" class="govuk-table__header" data-qa="sentence-col-date">Sentence start date</th>
+          <th scope="col" class="govuk-table__header" data-qa="sentence-col-length">Sentence length</th>
+        </tr>
+      </thead>
+      <tbody class="govuk-table__body ">
+          {% for case in model.cases %}
+              {% for sentenceModel in case.sentences %}
+              {% set sentence = sentenceModel.sentencesAndOffence %}
+              {% set offence = sentence.offence %}
+                  <tr class="govuk-table__row">
+                      <td class="govuk-table__cell" data-qa="sentence-{{ case.caseSequence }}-{{ sentence.lineSequence }}-title">Court case {{ case.caseSequence }}, Count {{ sentence.lineSequence }}</td>
+                      <td class="govuk-table__cell" data-qa="sentence-{{ case.caseSequence }}-{{ sentence.lineSequence }}-date">{{ sentence.sentenceDate | date('DD MMMM YYYY') }}</td>
+                      <td class="govuk-table__cell" data-qa="sentence-{{ case.caseSequence }}-{{ sentence.lineSequence }}-length">
+                      {% if sentenceModel.hasCustodialAndLicenseTerms() %}
+                          {{ sentenceLength(sentenceModel.imprisonmentTerm, 'Custodial term') }}
```

Comment:
possibly untested

Reply:
Already covered under - GET /view/:nomsId/calculation-summary/:calculationRequestId/printNotificationSlip?fromPage=view should generate page - CustodialTerms uses code IMP

<img width="475" alt="Screenshot 2024-05-16 at 14 13 50" src="https://github.com/ministryofjustice/calculate-release-dates/assets/165897256/68e6f39f-2ec5-4e2a-8afb-dca8736f1582">